signac/contrib/schema.py

```diff
@@ -136,6 +138,15 @@ def __str__(self):
     def __repr__(self):
         return "{}(<len={}>)".format(type(self).__name__, len(self))
 
+    def _repr_html_(self):
+        html = "{<br>"
+        for key in sorted(self._schema):
+            values = self._schema[key]
+            if values:
+                html += "<pre>  '{}': '{}',</pre>".format(key, self._fmt_values(values, 5))
+        html += "}"
+        return html
```

Comment:
I think Brandon has a good recommendation above. @vishav1771 It looks like this PR is re-creating the same formatting logic that is in `__str__`. I suggest the following code snippet -- I'm happy with this output (screenshot below).
```suggestion
        import html
        html_repr = "<strong>" + html.escape(repr(self)) + "</strong>"
        html_repr += "<pre>" + str(self) + "</pre>"
        return html_repr
```
![image](https://user-images.githubusercontent.com/3943761/79126229-2e4a1f80-7d65-11ea-9983-620a9f391de5.png)